code/lib/blocks/src/blocks/useTransformCode.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+
+import type { Args, StoryContext } from '@storybook/csf';
+
+type ReducedStoryContext = Omit<
+  StoryContext<any, Args>,
+  'abortSignal' | 'canvasElement' | 'step' | 'context'
+>;
+
+export function useTransformCode(
```

Comment:
a comment here to let authors know to keep it in sync with the same file in `preview-web`?

Reply:
Good call!